.github/ISSUE_TEMPLATE.md

```diff
@@ -1,23 +1,34 @@
-> Please provide us with the following information:
-> ---------------------------------------------------------------
-
-### OS?
-> Windows 7, 8 or 10. Linux (which distribution). Mac OSX (Yosemite? El Capitan?)
-
-
+<!--
+IF YOU DON'T FILL OUT THE FOLLOWING INFORMATION YOUR ISSUE MIGHT BE CLOSED WITHOUT INVESTIGATING
+-->
+### Bug Report or Feature Request (mark with an `x`)
+- [ ] bug report -> please search issues before submitting
+- [ ] feature request
+
+ <!-- if bug report, otherwise delete -->
```

Comment:
I think that you should add an additional empty line before this one, check the preview: https://github.com/Brocco/angular-cli/blob/aaa899e2de344f9d6000deac5593cd3c7f15dca2/.github/ISSUE_TEMPLATE.md